common/utils/constants.js

```diff
@@ -3,3 +3,15 @@ export const APD_TYPE = {
   HITECH: 'HITECH',
   MMIS: 'MMIS'
 };
+
+export const FUNDING_CATEGORY_TYPE = {
+  ddi: 'DDI',
+  mando: 'MO'
+};
+
+export const FUNDING_CATEGORY_LABEL_MAPPING = {
```

Comment:
what's the reason for having these vs just picking lower case or capitalized?

Reply:
I can switch it to also be uppercase that's more consistent